plugins/vpc-shared-eni/config/kubeapi.go

```diff
@@ -11,7 +11,8 @@
 // express or implied. See the License for the specific language governing
 // permissions and limitations under the License.
 
-// +build !disablekubeapi
+// +build enablekubeapi
```

Comment:
approving -- I'll note that this will break customers who are currently using the `disablekubeapi` build flag.